readme.md

```diff
@@ -48,6 +48,8 @@ Any `.node` files included will also support binary relocation.
             // defined that should not be emitted
             existingAssetNames: []
             wrapperCompatibility: false, // optional, default
+            writeMode: false // optional, default
```

Comment:
"`writeMode`" makes the option sound like you're switching from "read mode" to "write mode". 

Let's rename this to be `writeSupport` or `relocateWrites`.